sub-packages/bionemo-esm2/src/bionemo/esm2/scripts/train_esm2.py

```diff
@@ -432,6 +443,12 @@ def get_parser():
     parser.add_argument(
         "--wandb-group", type=str, default=None, help="A unique string shared by all runs in a given group"
     )
+    parser.add_argument(
+        "--wandb-job-type",
+        type=str,
+        default=None,
+        help="the type of run, which is useful when you're grouping runs together into larger experiments using group.",
+    )
     parser.add_argument(
```

Comment:
Would you please mirror these changes in `sub-packages/bionemo-llm/src/bionemo/llm/train.py` ? this may need to be an extension of the WandbConfig